tailwind.config.js

```diff
@@ -7,5 +7,22 @@ module.exports = {
       center: true,
       padding: '2rem',
     },
+    extend: {
+      fontFamily: {
+        primary: ['"Space Grotesk"', 'sans-serif'],
+        secondary: ['"JetBrains Mono"', 'sans-serif'],
+      },
+      colors: {
+        tertiary: '#01feb6',
+        'formal-accent': '#efecd6',
+        'primary-dark': '#0d0030',
+      },
+      boxShadow: {
+        button: '0px 0px 26px 0px rgba(1, 254, 182, 1)',
```

Comment:
Instead of `px` it's better to use `rem`. You may use [this service](https://nekocalc.com/px-to-rem-converter) to convert `px to rem`

Reply:
fixed, thanks